Sources/Filters/General/Calculator/test/testCalculator.js

```diff
@@ -93,3 +94,17 @@ test('Test vtkCalculator execution', (t) => {
 
   t.end();
 });
+
+test('make sure vtkCalculator does not crash with a vtkImageData input', (t) => {
+  const source = vtkImageGridSource.newInstance();
+  const filter = vtkCalculator.newInstance();
+
+  filter.setInputConnection(source.getOutputPort());
+  filter.setFormulaSimple(FieldDataTypes.POINT, ['scalars'], 'mask', (value) =>
+    value > 10 ? 1 : 0
+  );
+
+  source.update();
+  filter.update();
```

Comment:
should you also check if the output of the filter contains a data object with the same attributes as the input data object ?

Reply:
definitely. I updated the test